dot/rpc/subscription/subscription.go

```diff
@@ -6,54 +6,56 @@ import (
 	"strconv"
 )
 
-var errUknownParamSubscribeID = errors.New("invalid params format type")
-var errCannotParseID = errors.New("could not parse param id")
-var errCannotFindListener = errors.New("could not find listener")
-var errCannotFindUnsubsriber = errors.New("could not find unsubsriber function")
+const (
+	authorSubmitAndWatchExtrinsic  string = "author_submitAndWatchExtrinsic" //nolint
```

Comment:
Do you explicitly need to mention const as `string`?

Reply:
Yes, if I don't declare the string type for all consts Deepsource and `make lint` fails